templates/app/views/products/_product_taxons.html.erb

```diff
@@ -1,8 +1,8 @@
 <% if !product.taxons.blank? %>
   <section class="product-taxons">
-    <h2 class="product-taxons__title">
```

Comment:
There should only be one h1 heading on a page. Since this partial is able to be rendered on many pages including the PDP which has the product name as level heading I think we should leave it as h2 and maybe introduce a local that can be passed into he partial on order to be able to set the tag to something else.

Reply:
@tvdeyen In my personal opinion The secondary headings should be pulled from the resource description and not from arbitrarily placed tags. Is there any particular reason you want to keep this a heading other than style?